gr-CyberRadio/CMakeLists.txt

```diff
@@ -20,99 +21,85 @@
 ########################################################################
 # Project setup
 ########################################################################
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.8)
 project(gr-CyberRadio CXX C)
 enable_testing()
 
-#install to PyBOMBS target prefix if defined
+# Install to PyBOMBS target prefix if defined
 if(DEFINED ENV{PYBOMBS_PREFIX})
     set(CMAKE_INSTALL_PREFIX $ENV{PYBOMBS_PREFIX})
     message(STATUS "PyBOMBS installed GNU Radio. Setting CMAKE_INSTALL_PREFIX to $ENV{PYBOMBS_PREFIX}")
 endif()
 
-#select the release build type by default to get optimization flags
+# Select the release build type by default to get optimization flags
 if(NOT CMAKE_BUILD_TYPE)
    set(CMAKE_BUILD_TYPE "Release")
    message(STATUS "Build type not specified: defaulting to release.")
 endif(NOT CMAKE_BUILD_TYPE)
 set(CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE} CACHE STRING "")
 
-#make sure our local CMake Modules path comes first
+# Make sure our local CMake Modules path comes first
 list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_SOURCE_DIR}/cmake/Modules)
-list(INSERT CMAKE_MODULE_PATH 0 ${CMAKE_SOURCE_DIR}/gnuradio-volk-devel/usr/lib64/cmake/volk)
 
 # Set the version information here
-set(VERSION_INFO_MAJOR_VERSION 1)
-set(VERSION_INFO_API_COMPAT    0)
-set(VERSION_INFO_MINOR_VERSION 0)
-set(VERSION_INFO_MAINT_VERSION git)
-# -- If PACKAGE_VERSION is set from an external source, such as during Debian package
-#    builds, use it to set the version information.
-if(DEFINED PACKAGE_VERSION)
-    include(UnpackVersionInfo)
-    UNPACK_VERSION_INFO(PACKAGE_VERSION VERSION_INFO_MAJOR_VERSION VERSION_INFO_API_COMPAT
-                        VERSION_INFO_MINOR_VERSION VERSION_INFO_MAINT_VERSION)
-    #message(STATUS "UNPACK_VERSION_INFO ${PACKAGE_VERSION} ->")
-    #message(STATUS "* VERSION_INFO_MAJOR_VERSION=${VERSION_INFO_MAJOR_VERSION}")
-    #message(STATUS "* VERSION_INFO_API_COMPAT=${VERSION_INFO_API_COMPAT}")
-    #message(STATUS "* VERSION_INFO_MINOR_VERSION=${VERSION_INFO_MINOR_VERSION}")
-    #message(STATUS "* VERSION_INFO_MAINT_VERSION=${VERSION_INFO_MAINT_VERSION}")
-endif(DEFINED PACKAGE_VERSION)
+set(VERSION_MAJOR 1)
+set(VERSION_API   0)
+set(VERSION_ABI   0)
```

Comment:
Shouldn't the version be bumped?

Reply:
we actually version the package outside of cmake in our internal versioning.  It doesn't get pulled in here. We can edit that.